apps/web/lib/validators/escrow.ts

```
@@ -0,0 +1,86 @@
+    if (!milestone.title?.trim()) {
+        errors.push(`Milestone ${index + 1}: Title is required`);
+    }
+    if (!milestone.description?.trim()) {
+        errors.push(`Milestone ${index + 1}: Description is required`);
+    }
+    if (typeof milestone.amount !== "number" || milestone.amount <= 0) {
+        errors.push(`Milestone ${index + 1}: Amount must be a positive number`);
+    }
+    if (
+        !(milestone.dueDate instanceof Date) &&
+        !Date.parse(milestone.dueDate as any)
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)